src/applications/simple-forms/40-0247/pages/applicantAddress.js

```diff
@@ -9,11 +9,15 @@ import {
 export default {
   uiSchema: {
     'ui:title': (
-      <h3 className="vads-u-margin-y--0">
-        Tell us where we should send the certificate. If you want us to send
-        copies to another address, you can add an additional address later in
-        this form.
-      </h3>
+      <>
```

Comment:
looks like this might be a good candidate for:
uiSchema: {
...titleUI(titleText, descriptionText)

is that styling ok using titleUI?

Reply:
using tItleUI would be great here. All we would need to do is 

```
 ...titleUI(
      'Tell us where we should send the certificate.',
      'If you want us to send copies to another address, you can add an additional address later in this form.',
    ),
```

And all of the styling is already set. We would not need to add any other classes. We also can't add `<p>` tags inside of legends anyway, which is what makes titleUI great. It will wrap the instructional text in a span and add all of the proper classes. We did this in [21P-0837](https://github.com/department-of-veterans-affairs/vets-website/blob/016d7b30ee9520f521281e57a56c15d7a4a81ea9/src/applications/simple-forms/21P-0847/pages/deceasedClaimantPersonalInformation.js)